pkg/process/checks/process.go

```diff
@@ -44,6 +44,10 @@ var _ Check = (*ProcessCheck)(nil)
 
 var errEmptyCPUTime = errors.New("empty CPU time information returned")
 
+const (
+	ProcessDiscoveryHint int32 = 1 << iota // 1
```

Comment:
Not in generated go?

Reply:
No, as the mask is just an `int32` we have to define these constants manually.